docs/developer-setup/README.md

```diff
@@ -205,21 +202,25 @@ git submodule update --init --recursive
 ```sh
 ## Release
 cmake -B build
-cmake --build build --parallel 8
+cmake --build build --target install --parallel 8
 
 ## Debug
 cmake -B build-debug -D CMAKE_BUILD_TYPE=Debug
-cmake --build build-debug --parallel 8
+cmake --build build-debug --target install --parallel 8
 ```
```

Comment:
99% percent of the time we want to build **and** install, so I added `--target install` here.